service-front/module/Application/src/Controller/Authenticated/Lpa/DateCheckController.php

```diff
@@ -37,7 +40,7 @@ public function indexAction()
             'lpa-id' => $lpa->id
         ]));
 
-        $helperResult = DateCheckViewModelHelper::build($lpa);
+        $helperResult = ($this->dateCheckViewModelHelper)($lpa);
```

Comment:
Was there a reason we had to go with an invokable vs a static method? This line was confusing when I first saw it (but maybe that's just my brain getting used to PHP again 🤷).

Reply:
Yeah, if it's possible to use a static method, I think it would be easier to quickly parse what's going on here for anyone not familiar with the code (or language).